doc/src/manual/strings.md

```diff
@@ -371,6 +400,34 @@ julia> string(greet, ", ", whom, ".\n")
 "Hello, world.\n"
 ```
 
+An important to be aware of situation is when invalid UTF-8 strings are concatenated.
```

Comment:
Perhaps better as `A situation which is important to be aware of is when...`

Reply:
fixed. @fredrikekre thank you for a review 😄,